src/txmempool.h

```diff
@@ -851,28 +851,20 @@ class CCoinsViewMemPool : public CCoinsViewBacked
  * that are included in blocks in the new chain, and then process the remaining
  * still-unconfirmed transactions at the end.
  */
-
-// multi_index tag names
-struct txid_index {};
-struct insertion_order {};
-
 struct DisconnectedBlockTransactions {
```

Comment:
Nit: With private members and a ctor, I'd call this a `class` now. But that's arbitrary and I don't think we have any guidelines for it.

Reply:
True, changed